### ISSUE TITLE
privacy mode

### ISSUE THREAD

**Issue by adrelanos** [hexchat/hexchat#1578]
There is a package [xchat-improved-privacy](https://github.com/Whonix/xchat-improved-privacy) that has been developed under the Whonix umbrella. ([Whonix](https://www.whonix.org) is a privacy preserving Linux distribution.) That package is based on [The Tor Project's XChat wiki page](https://trac.torproject.org/projects/tor/wiki/doc/TorifyHOWTO/XChat#CompulsoryXChatPrivacySettings).

Essentially it's the following files which modify some settings for better privacy.

* https://github.com/Whonix/xchat-improved-privacy/blob/master/etc/skel/.xchat2/ctcpreply.conf
* https://github.com/Whonix/xchat-improved-privacy/blob/master/etc/skel/.xchat2/ignore.conf
* https://github.com/Whonix/xchat-improved-privacy/blob/master/etc/skel/.xchat2/servlist_.conf
* https://github.com/Whonix/xchat-improved-privacy/blob/master/etc/skel/.xchat2/xchat.conf

That package is unlikely to enter the Debian/Ubuntu/etc. repository, because writing to /etc/skel or user's home folder is not something they appreciate. They advice to put such features upstream.

Could you add a privacy mode please? Once turned on, all these settings are switched. Ideally, these could be enabled by an environment variable (`hexchat_privacy=1`) or by an `.d` configuration folder.

Another option might be to set these settings by default have have users opt-in for ctcp etc.

**Comment by TingPing** [comment]
I don't think that it makes much sense to have a flag for this. Most of the changes are the defaults in HexChat so the only things you would be doing is disabling CTCP, using generic Nick/Part/Etc, and configuring a proxy (which needs external setup anyway). I think it makes more sense to just document how to be more anonymous.

**Comment by adrelanos** [comment]
We would like to have all of that automatic and pre-configured in Whonix.

**Comment by TingPing** [comment]
Of the two solutions I do think a way for distributors to override default configs would be the better one as it is more flexible.

**Comment by adrelanos** [comment]
Yes. What about that .d config file mechanism? Then distributions could
just drop snippets.
